### PR TITLE
[feature][doc] Add offload metrics for broker and some minor fixes for tiered storage

### PR DESCRIPTION
**Pull request by momo-jun**
Doc updates for  #13833 

1. Add new offload metrics for broker.
    **Preview:**
image

2. In the Tiered Storage chapter, add a reference for offload metrics and fix minor issues for single-source purposes.
    a) add a link to offload metrics.
     **Preview:**
    image

    b) add a streamlined sentence and remove duplicate information in each offloader page.
     **Preview:**
    image

3. Fix style syntax issue for breaks in related doc pages.
4. Unify the expression for rates.

  
- [x] `doc`

### CODE REVIEW

**Comment by momo-jun** [comment]
@tjiuming @hangc0276 @Anonymitaet can you please review this PR as soon as #13833 is merged? Thank you.

**Review by Anonymitaet — DISMISSED** [review]

**Comment by Anonymitaet** [comment]
@momo-jun I've added `DO NOT MERGE`  to the PR title since we're waiting for the code PR to be merged first.

**Comment by momo-jun** [comment]
> @momo-jun I've added `DO NOT MERGE` to the PR title since we're waiting for the code PR to be merged first.

Makes sense. Thank you for doing this for me :)

File: site2/docs/reference-metrics.md [hunk 1]
```diff
@@ -496,6 +497,32 @@ All the jetty metrics are labelled with the following labels:
...
+All the offload metrics are labelled with the following labels:
+
+- *cluster*: `cluster=${pulsar_cluster}`. `${pulsar_cluster}` is the cluster name that you configured in `broker.conf`.
+- *namespace*: `namespace=${pulsar_namespace}`. `${pulsar_namespace}` is the namespace name.
+- *topic*: `topic=${pulsar_topic}`. `${pulsar_topic}` is the topic name.
+
+| Name | Type | Description |
+|---|---|---|
+| pulsar_ledgeroffloader_offloadError | Gauge | The number of offload requests that failed. |
+| pulsar_ledgeroffloader_offloadTime | Gauge | The total time (unit: millisecond) spent on the offload. |
```
[the remark this answers is not in the document]
Comment (momo-jun) [reply site2/docs/reference-metrics.md]: Thanks Yu. I've updated the related description.

**Comment by tjiuming** [comment]
The https://github.com/apache/pulsar/pull/13833 has merged already.
Please update the doc.

| NAME | TYPE | DESCRIPTION |
| -- | -- | -- |
| brk_ledgeroffloader_offload_error | COUNTER | Number of failed operations to offload |
| brk_ledgeroffloader_offload_rate | GAUGE | Rate of offload(bytes/s) |
| brk_ledgeroffloader_read_offload_error | COUNTER | Number of failed operations to read offload files |
| brk_ledgeroffloader_read_offload_rate | GAUGE | Rate of read entries from offload files(bytes/s) |
| brk_ledgeroffloader_write_storage_error | COUNTER | Number of failed operations to write to storage |
| brk_ledgeroffloader_read_offload_index_latency | SUMMARY | Latency of read index from offload files |
| brk_ledgeroffloader_read_offload_data_latency | SUMMARY | Latency of read data from offload files |
| brk_ledgeroffloader_read_ledger_latency | SUMMARY | Latency of read entries from bookkeeper |
| brk_ledgeroffloader_delete_offload_ops | COUNTER | Number of both successful and failed operations to delete offload |

**Comment by momo-jun** [comment]
@tjiuming @hangc0276 @Anonymitaet Can you pls review this doc PR? Thanks.

File: site2/docs/reference-metrics.md [hunk 2]
```diff
@@ -515,6 +516,27 @@ All the schema metrics are labelled with the following labels:
...
+All the offload metrics are labelled with the following labels:
+
+- *cluster*: `cluster=${pulsar_cluster}`. `${pulsar_cluster}` is the cluster name that you configured in `broker.conf`.
+- *namespace*: `namespace=${pulsar_namespace}`. `${pulsar_namespace}` is the namespace name.
+- *topic*: `topic=${pulsar_topic}`. `${pulsar_topic}` is the topic name.
+
+| Name                                           | Type    | Description                                                                     |
+|------------------------------------------------|---------|---------------------------------------------------------------------------------|
+| brk_ledgeroffloader_offload_error              | Counter | The number of failed operations to offload.                                     |
+| brk_ledgeroffloader_offload_rate               | Gauge   | The rate of offloading(bytes/s).                                                |
```
Comment (Anonymitaet) [inline site2/docs/reference-metrics.md:530]:
```suggestion
| brk_ledgeroffloader_offload_rate               | Gauge   | The rate of offloading (bytes/second).                                                |
```

File: site2/docs/reference-metrics.md [hunk 3]
```diff
@@ -515,6 +516,27 @@ All the schema metrics are labelled with the following labels:
...
+| brk_ledgeroffloader_read_offload_error         | Counter | The number of failed operations to read offload ledgers.                        |
+| brk_ledgeroffloader_read_offload_rate          | Gauge   | The rate of reading entries from offload ledgers(bytes/s).                      |
```
Comment (Anonymitaet) [inline site2/docs/reference-metrics.md:532]: same

File: site2/docs/reference-metrics.md (hunk 2 above)
Comment (momo-jun) [reply site2/docs/reference-metrics.md:530]: @Anonymitaet Makes sense. I will update accordingly.
Can you pls add it to the Pulsar writing style guide? There are many similar occurrences like "bytes/s".

Comment (Anonymitaet) [reply site2/docs/reference-metrics.md]: Sure, added here: https://docs.google.com/document/d/1lc5j4RtuLIzlEYCBo97AC8-U_3Erzs_lxpkDuseU0n4/edit#bookmark=id.esa4s02v3adl

**Comment by tjiuming** [comment]
LGTM @momo-jun

**Comment by tjiuming** [comment]
LGTM @momo-jun

**Review by Anonymitaet — APPROVED** [review]

**Pull request merged**